rust/operator-binary/src/controller.rs

```diff
@@ -280,6 +287,87 @@ impl ReconcilerError for Error {
     }
 }
 
+trait OpaLogLevel {
+    fn to_opa_literal(&self) -> String;
+}
+
+impl OpaLogLevel for LogLevel {
+    // based on https://www.openpolicyagent.org/docs/latest/cli/#options-10 opa has only log levels {debug,info,error}
+    fn to_opa_literal(&self) -> String {
+        match self {
+            LogLevel::TRACE => "debug",
+            LogLevel::DEBUG => "debug",
+            LogLevel::INFO => "info",
+            LogLevel::WARN => "error",
+            LogLevel::ERROR => "error",
+            LogLevel::FATAL => "error",
+            LogLevel::NONE => "error",
+        }
+        .into()
+    }
+}
```

Comment:
suggestion: It might make more sense to include this into `stackable-operator` directly. See here:
https://github.com/stackabletech/operator-rs/blob/f2f1b6bf52eae825735af13f050569be0f5e6a9f/crates/stackable-operator/src/product_logging/spec.rs#L332

Reply:
I wasn't sure since this seems like something product specific? But if it makes sense to have this in operator-rs I can move it there.

Reply:
https://github.com/stackabletech/operator-rs/pull/798